Performance tests still fragile

### Problem Description

The performance tests still fail from time to time.

### Expected behavior

- Performance tests pass reliably

### Additional context

The last tests I saw fail were 
```
-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/performance/test_performance.py::test_performance[OneHotEncoder-UniqueIntegerGenerator] - AssertionError: reverse_transform time: 0.0004069517476199996 > 0.0004
FAILED tests/performance/test_performance.py::test_performance[OneHotEncoder-UniqueStringGenerator] - AssertionError: reverse_transform time: 0.00040322616390999996 > 0.0004
```

--- a/rdt/performance/datasets/categorical.py
+++ b/rdt/performance/datasets/categorical.py
@@ -335,7 +335,7 @@ class UniqueIntegerGenerator(CategoricalGenerator):
                 'memory': 500000.0
             },
             'reverse_transform': {
-                'time': 0.0004,
+                'time': 0.0005,
                 'memory': 1000000.0,
             }
         }
@@ -362,7 +362,7 @@ class UniqueIntegerNaNsGenerator(CategoricalGenerator):
                 'memory': 1000000.0
             },
             'reverse_transform': {
-                'time': 0.0004,
+                'time': 0.0005,
                 'memory': 1000000.0,
             }
         }
@@ -389,7 +389,7 @@ class UniqueStringGenerator(CategoricalGenerator):
                 'memory': 500000.0
             },
             'reverse_transform': {
-                'time': 0.0004,
+                'time': 0.0005,
                 'memory': 1000000.0,
             }
         }

--- a/rdt/performance/datasets/pii.py
+++ b/rdt/performance/datasets/pii.py
@@ -36,7 +36,7 @@ class RandomStringGenerator(PIIGenerator):
                 'memory': 500.0
             },
             'reverse_transform': {
-                'time': 2e-05,
+                'time': 3e-05,
                 'memory': 1000.0,
             }
         }
@@ -63,7 +63,7 @@ class RandomStringNaNsGenerator(PIIGenerator):
                 'memory': 1000.0
             },
             'reverse_transform': {
-                'time': 2e-05,
+                'time': 3e-05,
                 'memory': 1000.0,
             }
         }

